dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -254,6 +263,10 @@ def __init__(self,
 
             output_classes = nest.map_structure(lambda _: ops.Tensor, output_dtypes)
 
+            self._setup_inputs(input_datasets, input_names, input_layouts)
+
+            # TODO(klecki): Inspect the graph in the pipeline and set proper defaults in the External Source
```

Comment:
Q: Do you need to really _inspect_ the graph? I think we should be able to get the external inputs by name.